ci-operator/jobs/openshift/microshift/openshift-microshift-main-periodics.yaml

```diff
@@ -13,14 +86,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-microshift-main-rebase-on-nightlies
-  reporter_config:
-    slack:
-      channel: '#microshift-alerts'
-      job_states_to_report:
-      - failure
-      - error
-      report_template: '<@here> :red_jenkins_circle: Job *{{.Spec.Job}}* ended with
-        *{{.Status.State}}*. <{{.Status.URL}}|View logs>'
```

Comment:
Do we want to remove that?

Reply:
well, this came out of running make jobs. Havent seen changes to the rebase so it must come from somewhere else. In a different PR I will set up slack alerts so I can add them back.

Reply:
https://docs.ci.openshift.org/docs/how-tos/notification/ doesn't explicitly say that, but I think it needs to be added manually to the job file.
[ci-operator's config](https://steps.ci.openshift.org/ci-operator-reference) does not include any `reporter`

These alerts were added by @eggfoobar in [this PR](https://github.com/openshift/release/pull/38115) - Egli could you chime in?

Reply:
I was just reading that too, let me add it back.

Reply:
Done.

Reply:
Yup! To my knowledge those still need to be added manually, thanks @pacevedom !